samples/bluetooth/hci_spi/prj.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_GPIO=y
+CONFIG_SPI=y
+CONFIG_MAIN_STACK_SIZE=512
+CONFIG_BT=y
+CONFIG_BT_HCI_RAW=y
+CONFIG_BT_MAX_CONN=20
+CONFIG_BT_TINYCRYPT_ECC=n
+CONFIG_BT_RX_BUF_LEN=255
```

Comment:
This shall be 258 for a payload of 251 bytes. Do see Data Lengths of 251 configured, why 255 bytes?